commcare_connect/opportunity/helpers.py

```diff
@@ -150,6 +150,14 @@ def get_annotated_opportunity_access_deliver_status(opportunity: Opportunity):
                     distinct=True,
                 ),
                 completed=F("approved") + F("rejected") + F("pending") + F("over_limit"),
+                duplicate=Count(
```

Comment:
@pxwxnvermx I am not sure if this is the right way to calculate duplicate count, in terms of the spec. Could you please validate this?

Reply:
Yeah, that looks good to me. Another way the duplicates can be calculated is to filter for CompletedWork which have `saved_completed_count` greater than 1.